frontend/ui/data/css_tweaks.lua

```diff
@@ -161,7 +161,48 @@ h1 + h6, h2 + h6, h3 + h6, h4 + h6, h5 + h6 { page-break-before: avoid !importan
                 id = "headings_align_center",
                 title = _("Center headings"),
                 css = [[h1, h2, h3, h4, h5, h6 { text-align: center !important; }]],
-                priority = 3, -- so it overrides the ones above
+                priority = 4, -- so it overrides the ones above
+            },
+        },
+        {
+            title = _("Text direction"),
+            {
+                title = _("About text direction"),
+                info_text = _([[
+Documents in languages like arabic or hebrew are to be rendered Right-To-Left. This does not only affects text layout, but also list items bullets and numbers (put on the right) and table layout (cells laid from right to left).
+
+Usually, the publisher has set the appropriate tags to force such rendering - but if he hasn't, or if you're reading plain text documents, you may want to manually enable it with these tweaks.
+Note that in the absence of such specifications KOReader will try to detect the language of each paragraph and set the approppriate rendering per paragraph.
+
+You may also want to enable, in top menu → Gear → Navigation →, Invert page turn taps and swipes.]]),
+                separator = true,
+            },
+            {
+                id = "body_direction_rtl",
+                title = _("Set document direction to RTL (Right-to-Left)"),
```

Comment:
Seems a bit wordy. Couldn't we suffice with something like this?
```suggestion
                title = _("Force RTL (Right-to-Left)"),
```

Or even (given the explanation item above):

```suggestion
                title = _("Force RTL"),
```

Reply:
Well, these tweaks do not "force" RTL or LTR. They just set it to the top node. It will be overriden if there are dir= below in the document.
I have added no tweak that "force" a direction (I guess they would need `* {direction: rtl !important;}` - and I don't think they would be useful).

Also, after all this, I don't quite like the wording "text direction", because, as we see, the text itself does not change direction (english stays Left-To-Right). "Document direction" is a bit less confusing, although it could give the idea it means "invert page turn taps and swipes" - that's why I added that little help text.
So, I don't know :) other suggestions ?

Reply:
How about just "RTL"? Or `Document direction RTL` (it doesn't need to be a sentence).

Reply:
That's fine :) Done.